src/components/accessories/appHeader/AppHeader.tsx

```diff
@@ -73,11 +87,38 @@ const AppHeader: FunctionComponent<TProps> = ({ breadcrumbMap }) => {
             <div className="appHeader__nav__item">{t("nav.pharmacy")}</div>
             <div className="appHeader__nav__item">{t("nav.ward")}</div>
             <div className="appHeader__nav__item">{t("nav.billing")}</div>
+            <div className="appHeader__nav__item">
+              <Tooltip title={t("login.signout")!} aria-label="sign out">
+                <ExitToAppIcon
+                  id="signout_icon"
+                  onClick={() => setOpenLogoutConfirmation(true)}
+                />
+              </Tooltip>
+            </div>
```

Comment:
Shouldn't be a appHeader__nav__item. This classes should be used in order to render the main menu.
Try to create a new div, maybe placing it near the languageSwitcher component and render it in this way (take a look at the span element):

```
<div className="userInfo__toolbar">
  <Tooltip title={t("login.signout")!} aria-label="sign out">
    <span>Welcome, <strong>{username}</strong></span>
    <ExitToAppIcon
      className="userInfo__toolbar_icon"
      id="signout_icon"
      onClick={() => setOpenLogoutConfirmation(true)}
    />
  </Tooltip>
</div>

```

Reply:
Y're right.